### ISSUE TITLE
jq command not found

### ISSUE THREAD

**Issue by oxr463** [rancherlabs/support-tools#78]
How should we handle situations where jq is missing from a node?

**Comment by dnoland1** [comment]
Can you be more specific on which support tool you are referring to? Or is this a general comment on making sure all support tools work when host does not have jq?

**Comment by dnoland1** [comment]
There are a few tricks you can use in kubectl to eliminate the need for jq. Example here - https://github.com/rancher/docs/pull/2944/files

**Comment by oxr463** [comment]
> Can you be more specific on which support tool you are referring to? Or is this a general comment on making sure all support tools work when host does not have jq?

Yes. I had someone run the Rancher leader script, but without jq, it had a blank space for the final line in the output where the leader should have been.

**Comment by dnoland1** [comment]
I guess that's my fault - https://github.com/rancherlabs/support-tools/pull/68#issuecomment-731424736

Could consider going back to the awk/tr version to remove dependence on jq.
